### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by ttahm3d**
- Added two helper classes in layout.css
- Created Modal and its documentation
- Made it responsive

### CODE REVIEW

**Review by krishanu2911 — APPROVED** [review]
Everything seems perfect to go No changes are required

File: css/components/modal.css [hunk 1]
```diff
@@ -37,8 +37,8 @@
```
Comment (vaibhav-mahalle) [inline css/components/modal.css:37]: commit message does not specify what kind of changes.

**Review by vaibhav-mahalle — COMMENTED** [review]
everything looks good.

File: css/components/modal.css (hunk 1 above)
Comment (ttahm3d) [reply css/components/modal.css:37]: i created entire thing from start. I could not come up with a better commit message within character limit.

Comment (vaibhav-mahalle) [reply css/components/modal.css:37]: understood.

**Review by vaibhav-mahalle — APPROVED** [review]

**Pull request merged**